tests/PKSim.Tests/Presentation/CompoundAlternativeTaskSpecs.cs

```diff
@@ -341,20 +343,20 @@ protected override void Context()
          _solubility_pKa_pH_Factor = new PKSimParameter().WithName(CoreConstants.Parameters.SOLUBILITY_P_KA__P_H_FACTOR);
          _solubility_pKa_pH_Factor.Formula = new ExplicitFormula("10 * (pH +1)");
          _solubility_pKa_pH_Factor.Formula.AddObjectPath(new FormulaUsablePath(new[] {ObjectPath.PARENT_CONTAINER, CoreConstants.Parameters.REFERENCE_PH}).WithAlias("pH"));
-         _compound.Add(refPh);
-         _compound.Add(solubilty);
-         _compound.Add(gainPerCharge);
-         _compound.Add(_solubility_pKa_pH_Factor);
-         _solubilityAlternative = new ParameterAlternative();
-         _solubilityAlternative.Add(refPh);
-         _solubilityAlternative.Add(solubilty);
-         _solubilityAlternative.Add(gainPerCharge);
-         _solubilityAlternative.Add(solubilityTable);
+
+         _compoundForCalculation = new Compound {refPh, solubilty, gainPerCharge, _solubility_pKa_pH_Factor};
+         _solubilityAlternative = new ParameterAlternative {refPh, solubilty, gainPerCharge, solubilityTable};
 
          var solubilityAlternativeGroup = new ParameterAlternativeGroup();
          solubilityAlternativeGroup.AddAlternative(_solubilityAlternative);
 
+
+         _originalRefPh = DomainHelperForSpecs.ConstantParameterWithValue(_refPhValue).WithName(CoreConstants.Parameters.REFERENCE_PH).WithDimension(DomainHelperForSpecs.NoDimension());
```

Comment:
This is the original parameter. Just testing that at the end of the test, it has the same value now because we are using a clone